roles/openshift_sanitize_inventory/tasks/__deprecations_logging.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Probably should do a couple things.

1)  Let's set a fact like openshift_deprecated_variables = True
2) At the end of the run, if openshift_deprecated_variables = True, print "You're using deprecated variables in your inventory.  Please compare your variables with DEPRECATED.yml" or something along those lines.
3) DEPRECATED.yml (or other filename, in reasonable place) is a ABC sorted list of all deprecated variables
4) Possibly build a script users can run to compare their inventory to items in deprecated.

This has the benefit of 1) each role is in charge of it's own deprecating 2) User is informed at end of run.